### PR TITLE
Compile to new better dll instead

### PR DESCRIPTION
**Pull request by w451**
Spotify tries to load dpapi.dll but doesn't actually use any functions it imports from it so you don't need to patch anything. Also isn't replaced when Spotify updates so it can be persistent.

### CODE REVIEW

File: Uninstall.bat [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
-echo oLink.WorkingDirectory = oWS.ExpandEnvironmentStrings^("!Esc_WorkLinkTarget!"^)
-echo oLink.Save
-)1>!cSctVBS!
-cscript !cSctVBS!
-DEL !cSctVBS! /f /q
-)
-) 
-
-echo Patch successfully removed
+@echo off
```
Comment (Nuzair46) [inline Uninstall.bat:1]: There doesn't seem to be any change in the code for this file. Can you revert this.

File: .gitignore [hunk 2]
```diff
@@ -20,3 +20,9 @@
 /src/Debug
 *.aps
 *.zip
+*.log
+*.obj
+*.tlog
+*.user
+*.vcxproj
```
Comment (Nuzair46) [inline .gitignore:27]: We need vcxproj file to build the dll for release. 
I would suggest removing any changes in the gitignore as for now

**Review by Nuzair46 — COMMENTED** [review]
@w451 Please take a look

File: .gitignore (hunk 2 above)
Comment (w451) [reply .gitignore:27]: Will do, sorry about that I am new to using github

Comment (w451) [reply .gitignore]: Let me know how this is I think I got it all right

**Comment by Nuzair46** [comment]
@w451, It looks like there are a lot of unintended changes in this PR.
Can you clear the git cache on your local and make the pr.
You could also delete this PR and open a new one with only the changes you were planning to raise.
Cannot merge at the current state.

**Pull request closed without merging**